modules/client/src/addresslistVoting/internal/constants.ts

```diff
@@ -15,7 +15,13 @@ export const AVAILABLE_FUNCTION_SIGNATURES: string[] = [
     "removeAddresses",
   ).format("minimal"),
 ];
-
+// TODO:
+// use abi from plugin metadata
+export const UPDATE_ABI: { [build: number]: MetadataAbiInput[] } = {
```

Comment:
this can be loaded from the plugin metadata stored on ipfs. So should this be a constant?